File: Beecrowd/C/1008.c
#include <stdio.h>

int main() {
    int numero, horasTrabalhadas;
    double valorHora;
    scanf("%d%d%lf", &numero, &horasTrabalhadas, &valorHora);
    printf("NUMBER = %d\n", numero);
    printf("SALARY = U$ %.2lf\n", horasTrabalhadas*valorHora);

    return 0;
}

File: Beecrowd/JavaScript/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const numero = parseInt(lines.shift());
const horasTrabalhadas = parseInt(lines.shift());
const valorHora = parseFloat(lines.shift());

console.log(`NUMBER = ${numero}`);
console.log(`SALARY = U$ ${(horasTrabalhadas*valorHora).toFixed(2)}`);
